src/index.js

```diff
@@ -87,6 +89,15 @@ export default function loader(content, map, meta) {
         : false,
     })
     .then((result) => {
+      // work-around for apparent bug in postcss that loses the paths to source
+      // files in existing map previously generated by postcss-loader:
+      if (result.map && map && map.file) {
+        /* eslint-disable no-underscore-dangle, no-param-reassign */
+        result.map._sourceRoot = path.dirname(map.file);
```

Comment:
Ideally `sourceRoot` should be `empty` and each sources relative to webpack context with unique prefix, we will improve source maps for css pipeline in near future, so please open issue with reproducible test repo and we will find solution